### PR TITLE
Directory Traversal File Include Payloads

### PR DESCRIPTION
**Pull request by n17h31sm**
This PR covers issue #567 

- [X] This PR handles the issue and requires no additional PRs.

**What did this PR accomplish?**

- Cut the size of the example payloads
- Include the [PayloadsAllTheThings](https://github.com/swisskyrepo/PayloadsAllTheThings) cheatsheets instead

### CODE REVIEW

**Comment by n17h31sm** [comment]
Hmm, if I use `+` instead of `%20` it is `404` for real. I don't have a problem accessing it with the urls that Actions thinks are broken. 
* If I don't encode the whitespace the link brakes.

**Comment by ThunderSon** [comment]
Safe to ignore for now, we'll review what's going on.

File: document/4-Web_Application_Security_Testing/05-Authorization_Testing/01-Testing_Directory_Traversal_File_Include.md [hunk 1]
```diff
@@ -54,48 +54,23 @@ Here are some examples of the checks to be performed at this stage:
...
 ```
 
-For the cookies example:
-
-```text
-Cookie: USER=1826cc8f:PSTYLE=../../../../etc/passwd
-```
-
-It's also possible to include files and scripts located on external website:
+Another  common example is including content from an external source:
```
Comment (kingthorin) [inline document/4-Web_Application_Security_Testing/05-Authorization_Testing/01-Testing_Directory_Traversal_File_Include.md:63]:
```suggestion
Another common example is including content from an external source:
```

File: document/4-Web_Application_Security_Testing/05-Authorization_Testing/01-Testing_Directory_Traversal_File_Include.md [hunk 2]
```diff
@@ -107,22 +82,11 @@ Each operating system uses different characters as path separator:
...
-  - `..%2f` represents `../`
-  - `%2e%2e%5c` represents `..\`
-  - `%2e%2e\` represents `..\`
-  - `..%5c` represents `..\`
-  - `%252e%252e%255c` represents `..\`
-  - `..%255c` represents `..\` and so on.
-- Unicode/UTF-8 Encoding (it only works in systems that are able to accept overlong UTF-8 sequences)
-  - `..%c0%af` represents `../`
-  - `..%c1%9c` represents `..\`
+You can find encoding techniques an ready to be used directory traversal payloads at [PayloadsAllTheThings - Directory Traversal](https://github.com/swisskyrepo/PayloadsAllTheThings/tree/master/Directory%20Traversal)
```
Comment (kingthorin) [inline document/4-Web_Application_Security_Testing/05-Authorization_Testing/01-Testing_Directory_Traversal_File_Include.md:87]:
```suggestion
You can find encoding techniques and ready to use directory traversal payloads at [PayloadsAllTheThings - Directory Traversal](https://github.com/swisskyrepo/PayloadsAllTheThings/tree/master/Directory%20Traversal)
```

File: document/4-Web_Application_Security_Testing/05-Authorization_Testing/01-Testing_Directory_Traversal_File_Include.md [hunk 3]
```diff
@@ -107,22 +82,11 @@ Each operating system uses different characters as path separator:
...
 
-There are other OS and application framework specific considerations as well. For instance, Windows is flexible in its parsing of file paths.
+#### Windows specific considerations
```
Comment (kingthorin) [inline document/4-Web_Application_Security_Testing/05-Authorization_Testing/01-Testing_Directory_Traversal_File_Include.md:89]:
```suggestion
#### Windows Specific Considerations
```

File: document/4-Web_Application_Security_Testing/05-Authorization_Testing/01-Testing_Directory_Traversal_File_Include.md [hunk 4]
```diff
@@ -54,48 +54,23 @@ Here are some examples of the checks to be performed at this stage:
 
 The next stage of testing is analyzing the input validation functions present in the web application. Using the previous example, the dynamic page called `getUserProfile.jsp` loads static information from a file and shows the content to users. An attacker could insert the malicious string `../../../../etc/passwd` to include the password hash file of a Linux/UNIX system. Obviously, this kind of attack is possible only if the validation checkpoint fails; according to the file system privileges, the web application itself must be able to read the file.
 
-To successfully test for this flaw, the tester needs to have knowledge of the system being tested and the location of the files being requested. There is no point requesting `/etc/passwd` from an IIS web server.
+**Note:** To successfully test for this flaw, the tester needs to have knowledge of the system being tested and the location of the files being requested. There is no point requesting `/etc/passwd` from an IIS web server.
```
Comment (kingthorin) [inline document/4-Web_Application_Security_Testing/05-Authorization_Testing/01-Testing_Directory_Traversal_File_Include.md:57]: @ThunderSon should this be:
```md
> Note: To successfully test for this flaw, the tester needs to have knowledge of the system being tested and the location of the files being requested. There is no point requesting `/etc/passwd` from an IIS web server.
```

**Review by kingthorin — CHANGES_REQUESTED** [review]

File: document/4-Web_Application_Security_Testing/05-Authorization_Testing/01-Testing_Directory_Traversal_File_Include.md (hunk 4 above)
Comment (ThunderSon) [reply document/4-Web_Application_Security_Testing/05-Authorization_Testing/01-Testing_Directory_Traversal_File_Include.md:57]: I am considering not to use `>` for notes. It's representation is coming off a bit too zoned out in the PDF. Let's keep it as is for this one, I'm reviewing this to follow on across the whole guide. Maybe have a smaller text, or change it a bit like in books.

**Review by kingthorin — APPROVED** [review]
Thanks

**Pull request merged**